themes/classic/_dev/css/components/products.scss

```diff
@@ -35,9 +35,9 @@
   color: $gray-darker;
 
   .manufacturer-logo {
-    height: 3.125rem;
+    max-height: 4rem;
     margin-bottom: 1rem;
-    border: none;
+    border: 2px solid $gray-light;
```

Comment:
I'm not sure if we need this. It's not really needed to fix the problem with responsiveness, right?

Reply:
It does improve it a bit. When it's on white background it would look a little off. Border fixes that and it's subtle shouldn't bother anyone
![obraz](https://user-images.githubusercontent.com/5007456/141959517-4166803f-5376-43cb-8754-eeea76bcf27f.png) vs ![obraz](https://user-images.githubusercontent.com/5007456/141959353-338fce00-a6fc-4b4d-85ad-6f9630cdfa8a.png)

Reply:
cc @NeOMakinG - I think we could give this the green light as it uses a variable for the color, right?

Reply:
```suggestion
    border: none;
```

Reply:
I think we kicked this border months ago, but not sure

It's a question for the @PrestaShop/product-team and @prestascott more particularly

I wouldn't mind myself, the border is more UX but I can't take the decision ;)

Reply:
Yes, there was border here before, but I believe it was wider and classic-blue, so that actually was too much.

Reply:
@Progi1984 if without border, then `border: none;` is not needed. I switched `img-thumbnail` class (which added border + unnecessary padding, transition: all and potentially border-radius) for `img-fluid` which only handles responsiveness, so you wouldn't need to remove border anymore.

Reply:
So... border or no border?

Reply:
We can remove the border :)